modules/monitoring/src/js/pnp/pnp.js

```diff
@@ -11,8 +11,8 @@ $(document).ready(function () {
 					{
 						data: {
 							param: match[1],
-							view: match[2],
-							source: match[3],
+							source: match[2],
+							view: match[3],
```

Comment:
Nice! Thanks for fixing my bug from a previous commit. :)

Reply:
;)